tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/composite_instruction.h

```diff
@@ -186,6 +195,11 @@ class CompositeInstruction
   iterator insert(const_iterator p, const value_type& x);
   iterator insert(const_iterator p, value_type&& x);
   iterator insert(const_iterator p, std::initializer_list<value_type> l);
+  template <class InputIt>
+  void insert(iterator pos, InputIt first, InputIt last)
```

Comment:
I think this should be based on looking at the one from std::vector:
``` c++
template <class InputIt>
interator insert(const_iterator pos, InputIt first, InputIt last)
{
  return container_.insert(pos, first, last);
}
```

Reply:
Fixed